substrate-node/pallets/pallet-smart-contract/src/tests.rs

```diff
@@ -3598,6 +3656,21 @@ fn prepare_service_consumer_contract() {
     ));
 }
 
+fn prepare_solution_provider_alice() {
```

Comment:
Would be good to open up the account in this function. You never know in the future who you might want to create a solution provider for.